recipes/glimpse-bio/build.sh

```diff
@@ -1,20 +1,29 @@
 #!/bin/bash
 
+COMMIT_VERS="b3ff92f"
+COMMIT_DATE="2022-12-07"
```

Comment:
I don't think we want to hard-code this, since then we have to remember to update this each time a version is updated, which is especially fraught when bioconda-bot auto-bumps.
```suggestion
export __COMMIT_ID__="$(git rev-parse --short HEAD)"
export __COMMIT_DATE__ ="$(git show -s --format="%ci" | cut -f 1 -d ' ')"
```